model/textparse/protobufparse.go

```diff
@@ -77,6 +78,9 @@ type ProtobufParser struct {
 
 	// The following are just shenanigans to satisfy the Parser interface.
 	metricBytes *bytes.Buffer // A somewhat fluid representation of the current metric.
+
+	// mainly used in formatOpenMetricsFloat.
```

Comment:
```suggestion
	// floatFormatBuf is used in formatOpenMetricsFloat.
```

- While this is not exported, I still think we should follow the Go styleguide for doc comments (start with the name you are commenting).
- It is not "mainly" used in in formatOpenMetricsFloat, but exclusively, isn't it?

In different news, this should be further up, or you have to change the previous doc comment for `metricBytes` (because `floatFormatBuf` is not "just shenanigans to satisfy the Parser interface").

Reply:
Accuracy is always worth it; thanks for your sharp eye.